application/android/app/src/main/java/com/group12/beabee/views/entities/TaskFragmentEdit.java

```diff
@@ -72,6 +88,17 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         etTitle.setText(taskDTO.title);
         etDescription.setText(taskDTO.description);
         cbIsDone.setChecked(taskDTO.isDone);
+        tvDeadline.setText(taskDTO.deadline);
+        //Button button = (Button) findViewById(R.id.btn_pickDate);
```

Comment:
You can add onclicklisteners both ways, but again commented out code

Reply:
I removed these lines. Thanks